requirements.txt

```diff
@@ -25,4 +25,4 @@ nbsphinx-link
 sphinx-gallery
 pyfar
 pre-commit
-ipympl
\ No newline at end of file
+ipympl
```

Comment:
Is there something missing here? Looks like a falsely managed merge conflict?

Reply:
Yep